eggplant/core/context_processors.py

```diff
@@ -4,9 +4,24 @@
 
 
 def coop_vars(request):
+    # This problem is solved in several CMS, like django-cms and Wagtail. But
+    # since we don't rely on them, here's a quick naive implementation that
+    # simply takes for granted that the first part of a URL is the language
+    # code.. so that's like a contract. Any page we render, we assume to be
+    # through i18n_patterns
+    LANGUAGE_CHOOSER = {}
+    for lang in settings.LANGUAGES:
+        path_components = request.path.split("/")
+        lang_path = ["", lang[0]] + path_components[2:]
```

Comment:
Is this failing gracefully when on paths that does not have any slashes in them?